src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -43,27 +43,28 @@ public MarkLettersPostedTask(LetterRepository repo,
     }
 
     public void run(LocalTime now) {
+        if (!ftpAvailabilityChecker.isFtpAvailable(now)) {
+            logger.info("Not processing letter postage report due to FTP downtime window");
+            return;
+        }
+
         logger.info("Started report processing job");
 
-        if (ftpAvailabilityChecker.isFtpAvailable(now)) {
-            ftpClient
-                .downloadReports()
-                .stream()
-                .map(parser::parse)
-                .forEach(parsedReport -> {
-                    insights.trackPrintReportReceived(parsedReport);
-                    parsedReport.statuses.forEach(this::updatePrintedAt);
+        ftpClient
+            .downloadReports()
+            .stream()
+            .map(parser::parse)
+            .forEach(parsedReport -> {
+                insights.trackPrintReportReceived(parsedReport);
+                parsedReport.statuses.forEach(this::updatePrintedAt);
 
-                    if (parsedReport.allRowsParsed) {
-                        logger.info("Report {} successfully parsed, deleting", parsedReport.path);
-                        ftpClient.deleteReport(parsedReport.path);
-                    } else {
-                        logger.warn("Report {} contained invalid rows, file not removed.", parsedReport.path);
-                    }
-                });
-        } else {
-            logger.info("Not processing due to FTP downtime window");
```

Comment:
logging remains the same, why this change??

Reply:
As method evolves the nesting goes deeper and deeper making the indentation to 4th level. This is not needed when you can break the process at the very beginning then continue evolving/reading method normally without additional nesting wrap.
Makes sense to keep small if/else blocks. Here main part is actually responsible for something greater and making the block first level is natural